### DISCUSSION TITLE
Maximally mixed state

### DISCUSSION THREAD

**Discussion by j1ver** [QuSpin/QuSpin#490] [Q&A] [answered]
I want to simulate some quench dynamics in the Fermi Hubbard Model from zero hopping to a finite value.
My system should initially be in the maximally mixed state at half filling, so I need to do the time evolution of the density matrix, since this state is not a pure one. The density matrix at lattice site i is
<img src="https://render.githubusercontent.com/render/math?math=\rho(t=0)_i=1/2( \uparrow><\uparrow| \text{plus} \downarrow><\downarrow| )">
I create the basis via 
spinful_fermion_basis_1d(L=L,Nf=(L//2,L//2))

and borrowed the function for the time-evolution from the Lindblad example.

My question is how to implement the inital density matrix? 
If I just use 1/H.Ns * diag(1) my state stays the same since this matrix commutes with the Hamiltonian.

**Comment by j1ver** [comment]
If i would use the full basis I think for one site the density matrix would simply be
rho_i=diag(0,1/2,1/2,0) 
and then I could use rho=tensorproduct_{lattice sites}rho_i

**Comment by weinbe58** [comment]
I’m a bit confused about the motivation of starting in the fully mixed state. Are you looking at the autocorrelation function of some observable?

**Reply by j1ver** [reply]
Kind of. I am interested in some correlation functions to see if they thermalize or not and the fully mixed state should be my mean-field background.

**Reply by j1ver** [reply]
This maximally mixed state, together with the spin staggered state, are both groundstates of the Fermi hubbard with zero hopping and I didnt want to bother with the staggering.

**Comment by mgbukov** [comment]
I just wanna mention in passing, that there is the `eom` variable of the `evolve` method in  the `hamiltonian` class to evolve a density matrix. This can be useful if you have a time-dependent generator of dynamics.

**Reply by j1ver** [reply]
Yes, thank you! I used the https://weinbe58.github.io/QuSpin/examples/example17.html#example17-label example to declare the von Neumann dynamics.

**Comment by j1ver** [comment]
I found an error of mine. The maximally mixed state is a different one from what I mentioned earlier. This also has to missing and double occupation as states and should have no time evolution, since it is just proportional to the unit matrix. 
But the state I mentioned earlier can be written as diag(0,1/2,1/2,0) for every lattice site and should have a time evolution (even in the on-site expectation value). This I cannot produce in QuSpin, everything is constant for me. Even the measurement of the double occupation yields me a 1/4, which is not correct.

**Comment by j1ver** [comment]
Is it possible to delete this? I found my mistake.

**Comment by mgbukov** [comment, marked as answer]
No worries -- discussions can't be closed so let's leave it as is :)
